WalletWasabi.Gui/Config.cs

```diff
@@ -27,6 +28,7 @@ public class Config : ConfigBase
 		public const int DefaultPrivacyLevelStrong = 50;
 		public const int DefaultMixUntilAnonymitySet = 50;
 		public const int DefaultTorSock5Port = 9050;
+		public const int DefaultJsonRpcServerPort = 18099;
```

Comment:
We should go with `37128` instead, as the wasabi ports had been chosen in a way to avoid port collision as much as possible: https://docs.wasabiwallet.io/building-wasabi/Ports.html